app/views/shared/_add_repos_list.html.slim

```diff
@@ -1,5 +1,7 @@
-- if repos.blank?
-  p None!
+- if repos.blank? || repos.count == 0
+  table.table.table-striped
+    tr
```

Comment:
I think we should use https://github.com/codetriage/codetriage/blob/master/lib/sorted_repo_collection.rb#L12 instead of count. Otherwise this looks :+1: